doc/getting_started.md

```diff
@@ -21,7 +21,7 @@ Although the most straightforward way to use Torque is through either the CartoD
   </body>
 ```
 
-For Torque to work with your table, you only need a username, the name of the table, and a CartoCSS string to style the map. Leaflet's method `addTo` adds the Torque layer to the map. `play` runs the animation with the options specified in the CartoCSS properties.
+Torque libraries are automatically generated when the HTML document is added. For Torque to work with your table, you only need a username, the name of the table, and a CartoCSS string to style the map. Leaflet's method `addTo` adds the Torque layer to the map. `play` runs the animation with the options specified in the CartoCSS properties.
```

Comment:
I'd change this by "torque dependencies are packaged within the torque.js library so only need to include that file" (this but written in proper english :)